mantle/platform/machine/do/cluster.go

```diff
@@ -97,7 +97,7 @@ func (dc *cluster) NewMachineWithOptions(userdata *conf.UserData, options platfo
 
 func (dc *cluster) vmname() string {
 	b := make([]byte, 5)
-	rand.Read(b)
+	rand.Read(b) //nolint
```

Comment:
Why is this function even being called if we aren't doing anything with the return?  Does removing this affect anything?

Reply:
Traditionally the `vmname` function inside clusters was used to provide randomized VM names with context to the cluster to have an identifier to which cluster (test) they belong to in the vm name. The API for the platform would call it inside `platform/api/<platform>` when creating the machines in the given platform.

Reply:
Thanks for the context. Make sense. I think the error is not vital to handle. So, just add nolint.